atomsci/ddm/pipeline/model_wrapper.py

```diff
@@ -1569,7 +1569,7 @@ def _create_output_transformers(self, model_dataset):
                 transformers: A list of deepchem transformation objects on response_col, only if conditions are met
         """
         # TODO: Just a warning, we may have response transformers for classification datasets in the future
-        if self.params.prediction_type=='regression' and self.params.transformers==True:
+        if self.params.prediction_type=='regression' and self.params.transformers:
```

Comment:
Same comment as before
```suggestion
        if self.params.prediction_type=='regression' and self.params.transformers is True:
```